skyportal/tests/api/test_sharing.py

```diff
@@ -127,69 +127,6 @@ def test_sharing_photometry_with_foreign_group(
     assert data["data"]["obj_id"] == public_source.id
 
 
-def test_sharing_photometry_shares_source(
```

Comment:
The behavior in this test is based on outdated logic for Object ownership and sharing and should no longer be included in the suite (automated checking of permissions caught this).